doc/development/tests/unit-test/general-guidelines.md

```diff
@@ -17,6 +17,25 @@ use `MyComponent.native.test.tsx` either if
 
 - the logic is shared between web and native
 - there is a file `MyComponent.native.tsx.`
+  -> in that case, you should use userEvent instead of fireEvent in case of pressing
+
+Example with userEvent :
+
+```tsx
+jest.useFakeTimers()
```

Comment:
Est ce qu'on veut ajouter une explication sur le besoin d'utiliser un useFakeTimers() ?

Reply:
Du coup tu penses que ça suffit pas : 
`If you don't use `jest.useFakeTimers()` you have this error : `It is recommended to use userEvent with fake timers\nSome events involve duration so your tests may take a long time to run.\nFor instance calling userEvent.longPress with real timers will take 500 ms.`` ?

Reply:
Si nickel, je n'avais pas vu 🙂